tests/jenkins/TestPublishDistributionBuildResults.groovy

```diff
@@ -177,10 +182,11 @@ class TestPublishDistributionBuildResults extends BuildPipelineTest {
         def componentCategory = "categoryA"
         def status = "success"
 
-        def result = script.generateAndAppendJson(indexName, component, componentRepo, componentRef, version, distributionBuildNumber, distributionBuildUrl, buildStartTime, rc, rcNumber, componentCategory, status)
```

Comment:
Why remove indexName?

Reply:
The `indexName` field is added as typo peter its not used in `generateAndAppendJson`, this was also removed in main lib `vars/publishDistributionBuildResults.groovy`. The index is created as `def indexName = "opensearch-distribution-build-results-${formattedDate}"`.